rust/template/src/api.rs

```diff
@@ -563,6 +567,16 @@ pub unsafe extern "C" fn ddlog_get_table_id(tname: *const raw::c_char) -> libc::
     }
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn ddlog_get_table_name(tid: libc::size_t) -> *const raw::c_char {
+    match HDDlog::get_table_name(tid) {
+        Ok(name) => ffi::CString::new(name)
+            .map(ffi::CString::into_raw)
+            .unwrap(),
```

Comment:
We are trying to get rid of `unwrap`s in C APIs as unwinding across ffi boundaries is undefined behavior (https://github.com/vmware/differential-datalog/issues/381). Can we return `NULL` here gracefully instead?

Reply:
I didn't read the chapter on unwrap yet, but this parallels the code for `profile` (minus the unwrap_or_else). Is the missing _else the problem?

Reply:
Yes, exactly. `unwrap` may panic whereas `unwrap_or_else` would evaluate to the "or else" part.